src/rpc/server.ts

```diff
@@ -149,6 +158,28 @@ export class Server {
     ): Promise<void> {
         reply.rpcStatus = "failed" // default to failed
         let requestId: number | null = null
+
+        const versionParsingResult = altoVersions.safeParse(
+            (request.params as any)?.version ?? this.defaultApiVersion
```

Comment:
a little confused on why we need the defaultApiVersion flag

Reply:
this is temporary because we the current requests are handled by /rpc where there is no version.